### PR TITLE
update vol_to_surf docstring

### PR DESCRIPTION
**Pull request by jeromedockes**
- no more "experimental" warning
- add minimal example
- add advice to use depth strategy if possible

### CODE REVIEW

**Review by bthirion — APPROVED** [review]
LGTM, mostly wiating for green CI

File: nilearn/surface/surface.py [hunk 1]
```diff
@@ -613,10 +615,20 @@ def vol_to_surf(img, surf_mesh,
     interpolated values are averaged to produce the value associated to this
     particular mesh vertex.
 
-    Warnings
+    Examples
     --------
-    This function is experimental and details such as the interpolation method
-    are subject to change.
+    When both the pial and white matter surface are available, the recommended
+    approach is to provide the `inner_mesh` to rely in the 'depth' sampling
```
Comment (ymzayek) [inline nilearn/surface/surface.py:621]:
```suggestion
    approach is to provide the `inner_mesh` to rely on the 'depth' sampling
```

**Review by ymzayek — APPROVED** [review]
See small typo, other than that looks good! Thx @jeromedockes !

**Comment by ymzayek** [comment]
Thx merging now

**Pull request merged**